app/src/main/java/corp/kairos/adamastor/Statistics/AppDetailStats.java

```diff
@@ -0,0 +1,60 @@
+package corp.kairos.adamastor.Statistics;
+
+import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+
+import corp.kairos.adamastor.AppDetail;
+
+
+public class AppDetailStats extends AppDetail {
```

Comment:
I could join the two classes, but I like this approach because join it will imply the instance variable statististics to be nullable. Also, when we get application usage stats we will need to set it to all the applications. In this approach, this is disposable class, that is initialized once used, and cleaned by the garbage collector. Tell me what you think

Reply:
So every time we go to the settings activity we'll be creating instances of this class for all apps?